### DISCUSSION TITLE
Declination (Angle deviation)

### DISCUSSION THREAD

**Discussion by Blederik** [osmandapp/OsmAnd#19671] [Maps Related]
Hello dear developer or members. This map navigation is the best ever i had. But I have problem with angle deviation. It has been going on for a long time and i tried whatever is available. I have accuration of sattelites aproximately 4m and magnetic compass with calman filter turned on (i tried also turn off and all variation). In case I rotate my device directly to item (flag) in real enviroment, it shows in Osmand map different direction. Deviation is horriblly big. For example u can see on the attached picture. Is it possible to find out where is problem? It occures on more mobile devices, not only me... Please, could you help me how can i calibrate it? Thank you in advance.
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/168139442/8da47b2f-a046-4f45-a9bb-49ecea9370a8"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by mankeil** [comment]
Have you tried calibrating your compass?

To do so, open Google Maps, click on the blue dot rappresenting your position then calibrate and follow the instructions.

Note that you can also calibrate the compass without opening Google maps by repeating the motion.

**Comment by Blederik** [comment]
It seems, it works more accurate now. But what is disturbing, i need to calibrate it everytime, before i want to see more accurate direction. Despite of this obstacle (movement of my hands, like i would be clown ), your advice helped me well. Thank you very much.
